python/lsst/afw/typehandling/testUtils.py

```diff
@@ -102,6 +103,13 @@ def getTestData(cls, keyClass):
         """
         return {keyClass(key): value for key, value in cls._testData.items()}
 
+    @staticmethod
+    def getValidKeys(mapClass):
+        try:
+            return {mapClass.dtype}
+        except AttributeError:
+            return {cls.dtype for cls in mapClass.__subclasses__()}
```

Comment:
Why is this a `set`? I would have thought there was exactly one valid key type.

Reply:
There can be multiple types if `mapClass` is a `TemplateMeta` class. Should I test for that directly? I thought this way was more Pythonic.

Reply:
Ah, that's fine. Maybe a comment about that difference here (because in my head this class is always a `GenericMap`, but that's clearly not true).

Reply:
Oh hey, this method doesn't have a docstring. Shame on me!